packages/component/src/Activity/StackedLayout.tsx

```diff
@@ -102,7 +101,12 @@ const StackedLayout: FC<StackedLayoutProps> = ({
   const fromUser = activity.from.role === 'user';
   const messageBackDisplayText: string = (isMessage && activity.channelData?.messageBack?.displayText) || '';
 
-  const activityDisplayText = isMessage ? messageBackDisplayText || activity.text : '';
+  const isLivestreaming = getActivityLivestreamingMetadata(activity);
```

Comment:
```suggestion
  const isLivestreaming = !!getActivityLivestreamingMetadata(activity);
```
Otherwise it's an object which may cause re-renders and would need memoization.